examples/screenshot_and_export_figure.py

```diff
@@ -85,7 +85,7 @@
 # The final one shows how the exported figure adapts to change in the layer extent.
 # In the second row are the screenshots, showing the fact that the entire canvas
 # is captured and that zoom is preserved.
-viewer.theme = 'dark'
+viewer.canvas_color = 'black'
```

Comment:
```suggestion
viewer.canvas_color = None
```

We want to have theme default color here